doc/man/scons.xml

```diff
@@ -564,30 +578,40 @@ and many of those supported by <application>cons</application>.
 <!-- for future development directions. Do not remove. -->
 
 <variablelist>
-  <varlistentry>
+  <varlistentry id="opt-b">
   <term><option>-b</option></term>
   <listitem>
 <para>Ignored for compatibility with non-GNU versions of &Make;</para>
   </listitem>
   </varlistentry>
 
-  <varlistentry>
+  <varlistentry id="opt-clean">
   <term>
     <option>-c</option>,
     <option>--clean</option>,
     <option>--remove</option>
   </term>
   <listitem>
-<para>Clean up by removing the selected targets,
+<para>Set <firstterm>clean</firstterm> mode.
+Clean up by removing the selected targets,
 well as any files or directories associated
 with a selected target through calls to the &f-link-Clean; function.
 Will not remove any targets which are marked for
 preservation through calls to the &f-link-NoClean; function.
+</para>
+<para>
+Files created directly by Python code in SConscript files,
+as opposed to work scheduled to builder actions during the build phase,
+are not affected by clean mode.  If it is important to clean up
+some other work in clean mode, steps need to be added to handle that.
+An SConscript file can determine which mode
+is active by querying &f-link-GetOption;, as in the call
```

Comment:
can't they just be added using `Clean()`?

Reply:
I thought that only worked for targets?

Reply:
manpage says "This specifies a list of files or directories which should be removed whenever the targets "... so should work with any file or directory.

Reply:
Okay I changed the wording to be more like the no-exec case, since this section already talked about using the Clean function that doesn't need adding.